flink-catalog-aws/flink-catalog-aws-glue/src/main/java/org/apache/flink/table/catalog/glue/operator/GlueTableOperator.java

```diff
@@ -0,0 +1,366 @@
+import software.amazon.awssdk.services.glue.model.GetTableResponse;
+import software.amazon.awssdk.services.glue.model.GetTablesRequest;
+import software.amazon.awssdk.services.glue.model.GetTablesResponse;
+import software.amazon.awssdk.services.glue.model.GlueException;
+import software.amazon.awssdk.services.glue.model.StorageDescriptor;
+import software.amazon.awssdk.services.glue.model.Table;
+import software.amazon.awssdk.services.glue.model.TableInput;
+import software.amazon.awssdk.services.glue.model.UpdateTableRequest;
+import software.amazon.awssdk.services.glue.model.UpdateTableResponse;
+
+import java.time.Instant;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.apache.flink.util.Preconditions.checkArgument;
+import static org.apache.flink.util.Preconditions.checkNotNull;
+
+/** Utilities for Glue Table related operations. */
+@Internal
+public class GlueTableOperator extends GlueOperator {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GlueTableOperator.class);
+
+    public GlueTableOperator(String catalogName, GlueClient glueClient, String glueCatalogId) {
+        super(catalogName, glueClient, glueCatalogId);
+    }
+
+    /**
+     * Create table in glue data catalog service.
+     *
+     * @param tablePath Fully qualified name of table. {@link ObjectPath}
+     * @param table instance of {@link CatalogBaseTable} containing table related information.
+     * @throws CatalogException on unexpected error happens.
+     */
+    public void createGlueTable(final ObjectPath tablePath, final CatalogBaseTable table)
+            throws CatalogException {
+
+        checkNotNull(tablePath, "tablePath cannot be null");
+        checkNotNull(table, "table cannot be null");
+        checkArgument(table instanceof ResolvedCatalogBaseTable, "table should be resolved");
+
+        final Map<String, String> tableProperties = new HashMap<>(table.getOptions());
+        String tableOwner = GlueUtils.extractTableOwner(tableProperties);
+        List<Column> glueTableColumns = GlueUtils.getGlueColumnsFromCatalogTable(table);
+        StorageDescriptor.Builder storageDescriptorBuilder =
+                StorageDescriptor.builder()
+                        .inputFormat(GlueUtils.extractInputFormat(tableProperties))
+                        .outputFormat(GlueUtils.extractOutputFormat(tableProperties));
+
+        TableInput.Builder tableInputBuilder =
+                TableInput.builder()
+                        .name(tablePath.getObjectName())
+                        .description(table.getComment())
+                        .tableType(table.getTableKind().name())
+                        .lastAccessTime(Instant.now())
+                        .owner(tableOwner)
+                        .viewExpandedText(GlueUtils.getExpandedQuery(table))
```

Comment:
since this is not a mandatory parameter in builder and we don't really support those yet, why do we set those?